sponsorship_compassion/model/move_line.py

```diff
@@ -9,23 +9,23 @@
 #
 ##############################################################################
 
-from openerp.osv import orm
-from openerp.tools.translate import _
+from openerp import api, exceptions, models, _
 
 
-class move_line(orm.Model):
+class move_line(models.Model):
     """ Adds a method to split a payment into several move_lines
     in order to reconcile only a partial amount, avoiding doing
     partial reconciliation. """
     _inherit = 'account.move.line'
 
-    def split_payment_and_reconcile(self, cr, uid, ids, context=None):
+    @api.multi
+    def split_payment_and_reconcile(self):

```

Comment:
This method won't be needed anymore in v8, so you can delete completely this file !